ui-tests/package.json

```
@@ -5,7 +5,9 @@
   "private": true,
   "scripts": {
     "start": "jupyter lab --config jupyter_server_test_config.py",
-    "test": "npx playwright test",
+    "start:timeline": "jupyter lab --config jupyter_server_test_config_timeline.py",
```

You can get rid of `ui-tests/jupyter_server_test_config_timeline.py` and do:
```suggestion
    "start:timeline": "jupyter lab --ServerApp.base_url=/api/collaboration/timeline/",
```